src/lib/devConsole.js

```diff
@@ -821,6 +821,64 @@ const getMeshArtifact = async (organizationId, projectId, workspaceId, workspace
 	}
 };
 
+/**
+ * Gets the enabled features for the tenant.
+ *
+ * This request bypasses the Dev Console and is sent directly to the Schema Management Service.
+ * As a result, we provide the orgCode instead of orgId since Dev Console usually performs the translation.
+ * The near-term goal is to stop using Dev Console as a proxy for all routes.
+ * @param organizationCode
+ * @returns {Promise<Object>}
+ */
+const getTenantFeatures = async organizationCode => {
+	const { accessToken, apiKey } = await getDevConsoleConfig();
+	const config = {
+		method: 'get',
+		url: `${SMS_BASE_URL}/organizations/${organizationCode}/features?API_KEY=${apiKey}`,
+		headers: {
+			'Authorization': `Bearer ${accessToken}`,
+			'x-request-id': global.requestId,
+		},
+	};
+
+	logger.info(
+		'Initiating GET %s',
+		`${SMS_BASE_URL}/organizations/${organizationCode}/features?API_KEY=${apiKey}`,
+	);
+
+	try {
+		const response = await axios(config);
+
+		logger.info('Response from GET %s', response.status);
+
+		if (response?.status === 200) {
+			logger.info(`Tenant Features : ${objToString(response, ['data'])}`);
+
+			return response.data;
+		} else {
+			let errorMessage = `Something went wrong: ${objToString(
+				response,
+				['data'],
+				'Unable to get tenant features.',
+			)}`;
+			logger.error(`${errorMessage}. Received ${response.status} response instead of 200`);
+
+			throw new Error(errorMessage);
+		}
+	} catch (error) {
+		if (error.response?.status === 404) {
```

Comment:
I don't think we should be returning a 404 if features are not found. It would just mean it is `disabled`

Reply:
That's true. I had originally done it for good measure. I'll remove it.

Reply:
Done